Web build fails: outdated web tests after plugin refactor

`./gradlew :web:build` fails during `tsc` with the following errors in
`web/src/shared/paste/__tests__/remove-html-image-plugin.test.ts`:

```
error TS2741: Property 'hashText' is missing in type '{}' but required in type 'PasteProcessContext'.
error TS2322: Type '{ html: string; }' is not assignable to type 'PasteItem'.
  Type '{ html: string; }' is missing the following properties from type 'HtmlPasteItem': type, identifiers, hash, size
error TS2322: Type '{ dataUrl: string; }' is not assignable to type 'PasteItem'.
  Type '{ dataUrl: string; }' is missing the following properties from type 'ImagesPasteItem': type, count, relativePathList, fileInfoTreeMap, and 3 more.
```

Root cause: two prior refactors changed the types the test relies on, but the test file was not updated:

- #4180 added a required `hashText` field to `PasteProcessContext`.
- `PasteItem` became a discriminated union where every variant requires `type`, `identifiers`, `hash`, and `size`.

The runtime logic of `RemoveHtmlImagePlugin` is unchanged; only the test fixtures need to be updated to match the current types.

--- a/web/src/shared/paste/__tests__/remove-html-image-plugin.test.ts
+++ b/web/src/shared/paste/__tests__/remove-html-image-plugin.test.ts
@@ -1,7 +1,8 @@
 import { describe, it, expect } from "vitest";
 import { isSingleImgInBody } from "../plugin/remove-html-image-plugin";
 import { RemoveHtmlImagePlugin } from "../plugin/remove-html-image-plugin";
-import { PasteTypeInt } from "@/shared/models/paste-item";
+import { PasteType, PasteTypeInt } from "@/shared/models/paste-item";
+import type { HtmlPasteItem, ImagesPasteItem } from "@/shared/models/paste-item";
 import type { TypedItem, PasteProcessContext } from "../plugin/paste-process-plugin";
 
 // ─── isSingleImgInBody ───────────────────────────────────────────────
@@ -59,15 +60,33 @@ describe("isSingleImgInBody", () => {
 
 describe("RemoveHtmlImagePlugin", () => {
   const plugin = new RemoveHtmlImagePlugin();
-  const ctx: PasteProcessContext = {};
+  const ctx: PasteProcessContext = { hashText: (s) => s };
 
   function htmlItem(html: string): TypedItem {
-    return { pasteType: PasteTypeInt.HTML, item: { html } };
+    const item: HtmlPasteItem = {
+      type: PasteType.HTML,
+      identifiers: [],
+      hash: "",
+      size: 0,
+      html,
+    };
+    return { pasteType: PasteTypeInt.HTML, item };
   }
+
+  const imagesItem: ImagesPasteItem = {
+    type: PasteType.IMAGE,
+    identifiers: [],
+    hash: "",
+    size: 0,
+    count: 1,
+    relativePathList: [],
+    fileInfoTreeMap: {},
+    dataUrl: "data:image/png;base64,AA==",
+  };
 
   const imageItem: TypedItem = {
     pasteType: PasteTypeInt.IMAGE,
-    item: { dataUrl: "data:image/png;base64,AA==" },
+    item: imagesItem,
   };
 
   it("removes HTML item when it is a single-img wrapper and IMAGE is present", () => {

